mongo/client.go

```diff
@@ -695,6 +699,52 @@ func (c *Client) configureCrypt(opts *options.AutoEncryptionOptions) error {
 	return err
 }
 
+// mongoLoggerFromEnv returns a mongologger set to the environment variables. If no logging environment variables
```

Comment:
The part about returning nil if no env vars are set is outdated. This version always returns a logger if there are no errors.